stockticker/stockticker.py

```diff
@@ -3,8 +3,8 @@
 import requests
 from requests import Timeout
 import simplejson as json
-from flask import Flask, render_template, url_for, jsonify
-from flask import  request # pylint: disable=W0611
+from flask import Flask, render_template, jsonify
+from flask import request, url_for # pylint: disable=W0611
```

Comment:
I think we can just drop `url_for` from the import. In my testing of your branch it definitely doesn't seem to be needed.